specs-go/config.go

```diff
@@ -487,6 +489,40 @@ type WindowsHyperV struct {
 	UtilityVMPath string `json:"utilityVMPath,omitempty"`
 }
 
+// VM contains information for virtual-machine-based containers.
+type VM struct {
+	// Hypervisor specifies hypervisor-related configuration for virtual-machine-based containers.
+	Hypervisor VMHypervisor `json:"hypervisor"`
```

Comment:
`hypervisor` is optional, so you'll want:

```go
Hypervisor *VMHypervisor `json:"hypervisor,omitempty"`
```

And similarly for `Image` below.